src/hnsw/scan.c

```diff
@@ -193,8 +195,14 @@ bool ldb_amgettuple(IndexScanDesc scan, ScanDirection dir)
         }
 
         ldb_dlog("LANTERN querying index for %d elements", k);
-        num_returned = usearch_search(
-            scanstate->usearch_index, vec, usearch_scalar_f32_k, k, scanstate->labels, scanstate->distances, &error);
+        num_returned = usearch_search_custom_ef(scanstate->usearch_index,
```

Comment:
Would be better to just add `ef` as optional parameter to `usearch_search` and if `ef != 0`, use the passed `ef`, otherwise use the `ef` from usearch header

Reply:
I would do this but C doesn't support optional parameters in functions. We can make `usearch_search` a variadic function but that seems like overkill.

The "`ef != 0` then use passed `ef`, otherwise use `ef` from usearch header" logic you mention is already added in the usearch library. So passing the `ef` from here, regardless of what it is, will lead to this desired functionality

Maybe just rename `usearch_search_custom_ef` to `usearch_search`? Or just to make it more readable, check in this code if `ef != 0` then use `usearch_search_custom_ef` and if not, use the original `usearch_search`?